near-plugins-derive/tests/pausable.rs

```diff
@@ -341,6 +347,25 @@ async fn test_not_paused_with_different_key() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_pause_view_method() -> anyhow::Result<()> {
+    let setup = Setup::new().await?;
+    let res = setup
+        .call_counter_modifier(&setup.unauth_account, "increase_1")
+        .await?;
+    assert_success_with_unit_return(res);
+    assert_eq!(setup.get_counter().await?, 1);
+
+    let res = setup
+        .pausable_contract
+        .pa_pause_feature(&setup.pause_manager, "get_counter")
+        .await?;
+    assert_success_with(res, true);
```

Comment:
We should also call the `get_counter` as a view method to ensure that we are able to do so.

Reply:
We already do a view call line below